src/components/SearchBar.js

```diff
@@ -1,6 +1,28 @@
-import React from "react";
+import React from "react"
+
+function SearchBar({
+  setIsAlphabetical,
+  setIsPrice,
+  handleSorting,
+  setFilterByType,
+  handleFilter,
+}) {
+  function handleRadioButtons(event) {
+    if (event.target.value === "Alphabetically") {
+      setIsAlphabetical(true)
+      setIsPrice(false)
+    } else if (event.target.value === "Price") {
+      setIsAlphabetical(false)
+      setIsPrice(true)
+    }
+    handleSorting()
+  }
```

Comment:
While this works, which is great, you could probably do some refactoring and reduce the complexity a bit by having a single `sort` state that could hold a string of the sort, or an empty string if there's no sort. Then, the sorting function could check for which sort is being used, like `sort === "price"`, and sort accordingly.